ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/StatementExecutorTest.java

```diff
@@ -449,6 +451,65 @@ public void shouldEnforceReferentialIntegrity() {
 
   }
 
+  @Test
+  public void shouldFailCreateAsSelectIfExceedActivePersistentQueriesLimit() {
+    // Given:
+    // Create streams and start two persistent queries
+    createStreamsAndTables();
```

Comment:
Can this not be done in `setUp`?

Also, rather than needing the comment, (I'm generally against comments as they invariably get out of date and indicate the code isn't readable), change the name of the function!

```suggestion
    createStreamsAndStartTwoPersistentQueries();
```